src/api.c

```diff
@@ -151,70 +166,21 @@ hub_retcode_t parse_at_command(void)
     return ARGUMENT_ERROR;
 }
 
-static void api_msg_timeout_handler(void)
+static uint32_t str_to_u32(char *num_as_string)
 {
-    g_msg_timeout = true;
-}
-
+    uint8_t digits = strlen(num_as_string);
+    uint32_t number = 0U;
+    number += num_as_string[0] - 48;
```

Comment:
w sumiee chyba zbędna  linia bo wystarczy fora od 0 zacząć

Reply:
To prawda. Tak to jest, jak się przekleja napisane kiedyś funkcje "do testów" :)